store/tikv/region_cache.go

```diff
@@ -300,6 +301,55 @@ func (c *RegionCache) GetRPCContext(bo *Backoffer, id RegionVerID) (*RPCContext,
 	}, nil
 }
 
+// GetFlashRPCContext returns RPCContext for a region must access flash store. If it returns nil, the region
+// must be out of date and already dropped from cache or not flash store found.
+func (c *RegionCache) GetFlashRPCContext(bo *Backoffer, id RegionVerID) (*RPCContext, error) {
+	ts := time.Now().Unix()
+
+	cachedRegion := c.getCachedRegionWithRLock(id)
+	if cachedRegion == nil {
+		return nil, nil
+	}
+	if !cachedRegion.checkRegionCacheTTL(ts) {
```

Comment:
how about using the txn-start-ts as the start time?

Reply:
The start time keeps the same as `GetRPCContext`. I have no idea what will cause if we use `txn-start-ts` instead of the Unix time.